### PR TITLE
Open EntitySearch on the page dashboard.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #4426 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/PostSearcherAutoSuggest.js [hunk 1]
```diff
@@ -99,14 +108,20 @@ export default function PostSearcherAutoSuggest( {
...
 		if ( ! searchTerm ) {
 			setResults( [] );
 		}
 	}, [ searchTerm ] );
 
+	useEffect( () => {
+		if ( currentEntityTitle ) {
+			setSearchTerm( currentEntityTitle );
+		}
+	}, [ currentEntityTitle, unifiedDashboardEnabled ] );
```
Comment (eugene-manuilov) [inline assets/js/components/PostSearcherAutoSuggest.js:123]: I believe `unifiedDashboardEnabled` is not needed for this `useEffect` hook, right?
```suggestion
	}, [ currentEntityTitle ] );
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @asvinb. PR looks good to me. Added one tiny comment after which I'll approve this PR. Could you please take a look?

File: assets/js/components/PostSearcherAutoSuggest.js (hunk 1 above)
Comment (asvinb) [reply assets/js/components/PostSearcherAutoSuggest.js]: You're right @eugene-manuilov . Most likely some leftover when I originally had the test for unifiedDashboard. Thanks for catching that 👍

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Thanks, @asvinb.

**Review by eugene-manuilov — APPROVED** [review]
LGTM! Thanks, @asvinb.

**Pull request merged**